README.md

```diff
@@ -59,11 +61,28 @@ cd ../bin
 ./epic --configpath /path/to/toml
 # send rpc command to daemon
 ./epic-cli [OPTIONS] [COMMAND]
+# type ./epic-cli --help to see the options
 ```
 
-We provide a default [config.toml](./config.toml), where seed address for the testnet is provided. You may customize a number of things, e.g., how to use spdlog, by editing it.
+A default [config.toml](./config.toml) with seed address for the testnet is provided. You may customize a number of things, e.g., how to use `spdlog`, by editing the config file.
+
+### Mining
+
+Once a daemon is started, you may start the mining process
+
+```bash
+# 1. set pass phrase
+./epic-cli set-passphrase
+# 2. login
+./epic-cli login
+# 3. (optional) create transactions
+# ./epic-cli create-randomtx 50000
```

Comment:
```suggestion
./epic-cli create-randomtx 1
```

for creating the first registration tx. This is not optional but a must. We will soon have a command for this purpose only.

Reply:
Good point